users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from users.views import UserView
+
+urlpatterns = [
+    path("/info", UserView.as_view()),
```

Comment:
```suggestion
    path("", UserView.as_view()),
```
1. /users 만으로도 충분할 것 같습니다.
2. /users/information 만약에 정보라는 것을 붙일 거면 줄이지 말고 사용해 주세요.

Reply:
""로 수정완료했습니다!